src/OrbitBase/ThreadPoolTest.cpp

```diff
@@ -60,7 +60,7 @@ TEST(ThreadPool, QueuedActionsExecutedOnShutdown) {
   {
     absl::MutexLock lock(&mutex);
     for (size_t i = 0; i < kNumberOfActions; ++i) {
-      thread_pool->Schedule([&]() {
+      (void)thread_pool->Schedule([&]() {
```

Comment:
well, that does not look good :) Maybe have Future<void> without nodiscard? I think this means we want to apply nodiscard to class and omit it for the method.

Reply:
Yes, I can do that.

Reply:
Done.